<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAB_MainWindow</class>
 <widget class="QMainWindow" name="DAB_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DAB_MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonConnexion">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMessage">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>260</y>
      <width>531</width>
      <height>24</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelEtat">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat de la connexion</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMessage">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Message de la banque :</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxOperation">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>631</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Opérations envoyées à la banque</string>
    </property>
    <widget class="QPushButton" name="pushButtonEnvoi">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>150</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Envoi</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>124</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButtonCompte">
        <property name="text">
         <string>Numéro de compte</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelMontant">
        <property name="text">
         <string>Montant</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>251</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEditNumero">
        <property name="text">
         <string>1234</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditMontant"/>
      </item>
     </layout>
    </widget>
    <widget class="QRadioButton" name="radioButtonSolde">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>50</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Solde</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonRetrait">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>80</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Retrait</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonDepot">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>110</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Dépot</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>56</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelAdresse">
       <property name="text">
        <string>Adresse :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPort">
       <property name="text">
        <string>Port :</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>70</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEditAdresse">
       <property name="text">
        <string>172.18.58.104</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPort">
       <property name="text">
        <string>8888</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listWidgetEtat">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>100</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
